packages/concerto-core/lib/modelutil.js

```diff
@@ -202,6 +202,18 @@ class ModelUtil {
         return (typeDeclaration !== null && typeDeclaration.isEnum());
     }
 
+    /**
+     * Returns the true if the given field is an map type
+     * @param {Field} field - the string
+     * @return {boolean} true if the field is declared as an map
+     * @private
+     */
+    static isMap(field) {
+        const modelFile = field.getParent().getModelFile();
+        const typeDeclaration = modelFile.getType(field.getType());
+        return (typeDeclaration !== null && typeDeclaration?.isMapDeclaration?.());
```

Comment:
`typeDeclaration !== null` is redundant given the nullish check

Reply:
done+cleaned up the other expressions while there.